random.js

```diff
@@ -0,0 +1,76 @@
+var slideIndex = 0;
+var slides = document.getElementsByClassName("mySlides");
+var dots = document.getElementsByClassName("slide-inactive");
+var i;
+var userClick = false;
+let myTimeout;
+
+function currentSlide(n){
+    showSlides(slideIndex = n);
+}
+
+let showSlides = n => {
+    slideIndex > slides.length ? slideIndex = 1 : false;
+    n < 1 ? slideIndex = slides.length : false;
```

Comment:
Don't use ternary expressions to assign variables. I would instead just replace with `if` 

```
if (n < 1) {
  slideIndex = slides.length;
}
```